app/webpacker/scripts/sort-by.js

```diff
@@ -0,0 +1,24 @@
+const initSortBy = ({sortby_form_id, sortby_form_select_class}) => {
+  const sortby_form = document.getElementById(sortby_form_id);
+  if (!sortby_form) return;
+
+  const sort_by_form_select = sortby_form.getElementsByClassName(sortby_form_select_class)[0];
+
```

Comment:
Can we reduce the space here and bottom of the file

Reply:
Done